### PR TITLE
fix using the same test/suite name when generating diffs

### PR DESCRIPTION
**Pull request by aluedeke**
with sending the test and suite name as meta data for the snapshots, the original and modified checks where not using the same baseline anymore. So diffs without baseline instead of changed diffs are generated.

We should adjust the java example to test the same as the wdio example in the future.

### CODE REVIEW

File: wd-java-testng/src/test/java/com/example/InventoryTest.java [hunk 1]
```diff
@@ -47,6 +46,11 @@ void checkInventoryPageLooksTheSame() {
         var inventoryPage = new InventoryPage(driver);
         inventoryPage.open();
 
+        // this is here to generate a diff to better demonstrate sauce visual
+        if(System.getProperty("modified") != null) {
```
Comment (kb-kerem) [inline wd-java-testng/src/test/java/com/example/InventoryTest.java:50]: I think we can use dotenv instead to be consistent

File: wd-java/src/test/java/com/example/InventoryTest.java [hunk 2]
```diff
@@ -46,6 +46,10 @@ void checkInventoryPageLooksTheSame() {
 
         var inventoryPage = new InventoryPage(driver);
         inventoryPage.open();
+        // this is here to generate a diff to better demonstrate sauce visual
+        if(System.getProperty("modified") != null) {
```
Comment (kb-kerem) [inline wd-java/src/test/java/com/example/InventoryTest.java:50]: dotenv comment applies here too

File: wd-java-testng/src/test/java/com/example/InventoryTest.java (hunk 1 above)
Comment (kb-kerem) [inline wd-java-testng/src/test/java/com/example/InventoryTest.java:50]:
```suggestion
        if(dotenv.get("modified") != null) {
```

File: wd-java/src/test/java/com/example/InventoryTest.java (hunk 2 above)
Comment (kb-kerem) [inline wd-java/src/test/java/com/example/InventoryTest.java:50]:
```suggestion
        if(dotenv.get("modified") != null) {
```

File: wd-java-testng/src/test/java/com/example/InventoryTest.java (hunk 1 above)
Comment (aluedeke) [reply wd-java-testng/src/test/java/com/example/InventoryTest.java:50]: this needs to be set on the terminal by end users and setting it on the terminal is different between windows and mac. so going for system properties was more straight forward, does this make sense?

Comment (kb-kerem) [reply wd-java-testng/src/test/java/com/example/InventoryTest.java:50]: yes, sorry for the confusion

**Review by kb-kerem — APPROVED** [review]

**Pull request merged**